src/legacy/views/secondary-bar/tests/use-sync-data-handler.test.tsx

```diff
@@ -6,26 +6,31 @@
 import React, { ReactElement, ReactNode } from 'react';
 
 import { renderHook } from '@testing-library/react';
-import { SoapNotify, useRefresh } from '@zextras/carbonio-shell-ui';
 import { useFolderStore, folderWorker } from '@zextras/carbonio-ui-commons';
+import { SoapNotify } from '@zextras/carbonio-ui-soap-lib';
 import { http } from 'msw';
 
-import { useSyncDataHandler } from 'legacy/views/secondary-bar/use-sync-data-handler';
+import { useInfoRefresh, useSync } from '../../../../../__mocks__/@zextras/carbonio-ui-soap-lib';
 import { getSetupServer } from '@jest-setup';
-import { useNotify } from '@test-utils/carbonio-shell-ui/carbonio-shell-ui';
 import { generateFolder } from '@test-utils/folders/folders-generator';
 import { handleGetFolderRequest } from '@test-utils/network/msw/handle-get-folder';
 import { handleGetShareInfoRequest } from '@test-utils/network/msw/handle-get-share-info';
+import { useSyncDataHandler } from 'legacy/views/secondary-bar/use-sync-data-handler';
 
 function getWrapper() {
 	// eslint-disable-next-line react/display-name
 	return ({ children }: { children: ReactNode }): ReactElement => <>{children}</>;
 }
 
 function mockSoapRefresh(mailbox: number): void {
-	(useRefresh as jest.Mock).mockReturnValue({
-		mbx: [{ s: mailbox }]
-	});
+	const result = {
+		mbx: [{ s: mailbox }] satisfies [{ s: number }]
+	};
+	jest.mocked(useInfoRefresh).mockReturnValue(result);
```

Comment:
Since useInfoRefresh and useSync are already mocked the wrapping through jest.mocked is redundant
```suggestion
	useInfoRefresh.mockReturnValue(result);
```